k8s/namespaces/jenkins/jenkins.yaml

```diff
@@ -472,7 +472,413 @@ spec:
+                      - gitHubApp:
+                          appID: "60878"
+                          description: "GitHub APP Key - rse"
+                          id: "hmcts-jenkins-rse"
+                          privateKey: "${hmcts-jenkins-rse-ghapp}"
+                      - gitHubApp:
+                          appID: "72816"
+                          description: "GitHub APP Key - unspec"
+                          id: "hmcts-jenkins-unspec"
+                          privateKey: "${hmcts-jenkins-unspec-ghapp}"
+                      - gitHubApp:
+                          appID: "62005"
+                          description: "GitHub APP Key - em"
+                          id: "hmcts-jenkins-em"
+                          privateKey: "${hmcts-jenkins-em-ghapp}"
+                      - gitHubApp:
+                          appID: "69800"
+                          description: "GitHub APP Key - hmi"
+                          id: "hmcts-jenkins-hmi"
+                          privateKey: "${hmcts-jenkins-hmi-ghapp}"
+                      - gitHubApp:
+                          appID: "75169"
+                          description: "GitHub APP Key - wa"
+                          id: "hmcts-jenkins-wa"
+                          privateKey: "${hmcts-jenkins-wa-ghapp}"
+                      - gitHubApp:
+                          appID: "79716"
+                          description: "GitHub APP Key - fact"
+                          id: "hmcts-jenkins-fact"
+                          privateKey: "${hmcts-jenkins-fact-ghapp}"
+                      - gitHubApp:
+                          appID: "90741"
+                          description: "GitHub APP Key - nfdiv"
+                          id: "hmcts-jenkins-nfdiv"
+                          privateKey: "${hmcts-jenkins-nfdiv-ghapp}"
+          jobs: |
+            jobs:
+              - url: https://raw.githubusercont ent.com/hmcts/cnp-jenkins-config/master/jobdsl/organisations-beta.groovy
+
+      sidecars:
+        configAutoReload:
+          # If enabled: true, Jenkins Configuration as Code will be reloaded on-the-fly without a reboot.  If false or not-specified,
+          # jcasc changes will cause a reboot and will only be applied at the subsequent start-up.  Auto-reload uses the Jenkins CLI
+          # over SSH to reapply config when changes to the configScripts are detected.  The admin user (or account you specify in
+          # master.adminUser) will have a random SSH private key (RSA 4096) assigned unless you specify adminSshKey.  This will be saved to a k8s secret.
+          resources:
+            limits:
+              cpu: 100m
+              memory: 100Mi
+            requests:
+              cpu: 50m
+              memory: 50Mi
+          env:
+            - name: REQ_TIMEOUT
+              value: "60"
+      ingress:
+        enabled: true
+        # For Kubernetes v1.14+, use 'networking.k8s.io/v1beta1'
+        apiVersion: "networking.k8s.io/v1beta1"
```

Comment:
based on https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/templates/jenkins-controller-ingress.yaml#L3-L9 looks like this should be removed as it's ignored and done via version comparison